### PR TITLE
Tutorial: Coffee machine

### PR DESCRIPTION
**Pull request by amolenaar**
### PR Type
What kind of change does this PR introduce?

- [ ] Bug fix
- [ ] Feature
- [ ] Chore (refactoring, formatting, local variables, other cleanup)
- [x] Documentation content changes

### What is the new behavior?

Provide a simple example: develop a coffee machine.

Goals:

* Work with some different types of diagrams (static/dynamic)
* Reap some benefits from modeling: component interfaces, risk analysis

### CODE REVIEW

**Comment by danyeaw** [comment]
@amolenaar I think this tutorial is ready now, would you mind giving it a review?

File: docs/index.rst [hunk 1]
```diff
@@ -43,6 +44,13 @@ Gaphor has excellent integration with :doc:`Sphinx <sphinx>` and :doc:`Jupyter n
    stereotypes
    merge_conflicts
 
+.. toctree::
+   :caption: Getting Started
+   :hidden:
+
+   getting_started
+   first_model
+
```
Comment (amolenaar) [inline docs/index.rst:53]: This toctree can be removed.

File: docs/coffee_machine_wrapup.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+As they worked, the ants encountered numerous challenges. They had to ensure
+that the machine was safe, efficient, and easy to use, all while meeting the
+unique needs of their feline client. But with their deep understanding of
+systems engineering and their commitment to key principles and concepts, they
+were able to overcome these challenges and design an exceptional espresso
+machine.
+
+In the end, Cappuccino was thrilled with the machine, which worked flawlessly
+and was a big hit with his customers. He was so impressed with the ants' work
+that he offered them a long-term contract to design all of his cafe's systems.
```
Comment (amolenaar) [inline docs/coffee_machine_wrapup.md:19]:
```suggestion
that he offered them a long-term contract to design all of his café's systems.
```

cafe -> café.

File: docs/coffee_machine_concept.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+## Domain Diagram
+
+A domain diagram is a graphical representation of the concepts, terms, and
+relationships within a specific domain. In the case of a coffee shop, a domain
+diagram could represent the key elements and relationships within the coffee
+shop domain.
+
+The following is a domain diagram that builds upon the context diagram with
+additional blocks:
+- Barista
```
Comment (amolenaar) [inline docs/coffee_machine_concept.md:90]: The Barista is an actor, right? Is it also a Block?

Comment (danyeaw) [reply docs/coffee_machine_concept.md:90]: Yes, although sometimes I have seen using a Block as the actor itself: https://mbse4u.com/2012/03/20/the-death-of-the-actor/

I'm hoping SysML v2 cleans this up a bit.

**Comment by amolenaar** [comment]
This is a very nice tutorial. I'm really tempted to fix item flows for associations and internal block diagrams (visualize a block as a diagram?)

**Comment by danyeaw** [comment]
@amolenaar Yup, I think us working on those couple of things should be high priority

**Pull request merged**